rust/stackablectl/src/cmds/operator.rs

```diff
@@ -327,9 +337,8 @@ async fn install_cmd(args: &OperatorInstallArgs, cli: &Cli) -> Result<String, Cm
             )
             .context(HelmSnafu)?;
 
-        // TODO (@NickLarsenNZ): Send this to the progress handler instead of using println
-        // info!(%operator, "Installed operator");
-        println!("Installed {} operator", operator);
+        info!(%operator, "Installed operator");
+        indicatif_println!("Installed {} operator", operator);
```

Comment:
We should try to avoid unnamed interpolations:

```suggestion
        indicatif_println!("Installed {operator} operator");
```

Reply:
https://github.com/stackabletech/stackable-cockpit/pull/376/commits/6d80e341512eca4abfc65bc610b112ae5f6fac58